main.tf

```diff
@@ -239,23 +335,10 @@ JSON
       ),
     )
     nlb_effect = var.default_allow || var.allow_nlb ? "Allow" : "Deny"
-    nlb_resources = length(var.nlb_accounts) > 0 ? jsonencode(
-      sort(
-        formatlist(
-          format(
-            "arn:aws:s3:::%s/%s/AWSLogs/%%s/*",
-            var.s3_bucket_name,
-            var.nlb_logs_prefix,
-          ),
-          var.nlb_accounts,
-        ),
-      ),
-      ) : jsonencode(
-      format(
-        "arn:aws:s3:::%s/%s/AWSLogs/%s/*",
-        var.s3_bucket_name,
-        var.nlb_logs_prefix,
-        data.aws_caller_identity.current.account_id,
+    nlb_resources = jsonencode(
+      formatlist(
+        format("arn:aws:s3:::%s/%%s/*", var.s3_bucket_name),
+        var.nlb_logs_prefixes,
```

Comment:
It looks like you're getting rid of the AWSLogs prefix too? Won't that break some existing functionality for people?

Reply:
The `AWSLogs` path component would be handled in the caller module.  I added an upgrade path note to the README.md.

Reply:
I still don't understand why we're getting rid of the nlb_accounts parameter and instead making people do format tricks.

Reply:
So the way our NLB (and ALB) terraform modules log to S3, they prefix the name of the load balancer, before the account id, so this current module version does not support that, which is why we need this PR.  The current version only supports one log prefix for all NLBs,.